composer.json

```diff
@@ -31,7 +31,13 @@
     }
   ],
   "keywords": [
-    "bedrock", "composer", "roots", "wordpress", "wp", "wp-config", "pantheon"
```

Comment:
I'm assuming that these changes are something a linter did - they seem to be an automated "put each value on its own line" sort of thing, which makes sense to me.

Reply:
Yes, I should have noted that in side-effects. I couldn't get my editor to _not_ reformat the json lol